src/ci/run.sh

```diff
@@ -51,7 +51,7 @@ fi
 # all test suites have all their deps in there (just the main bootstrap) so we
 # have the ability to disable this flag
 if [ "$NO_VENDOR" = "" ]; then
-  RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-vendor"
+  # RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-vendor"
```

Comment:
Because I'm not sure yet what's supposed to happen. I guess I should remove it if we want to cache `.cargo` instead through some other means.